.samples/neo42PackageConfigJson.MD

```diff
@@ -637,8 +648,8 @@ default value: false
 switch to control the behavior for using the defined parameters in value "InstPara" (valid are only 'true' and 'false')
 note: by default a logfile always will be appended to used parameters (if supported/applied by setup), even if parameters will be replaced!
 ```
-- **true** : appends the parameters to this one get from the corresponding msi parameters entry in ADT config file 'AppDeployToolkitConfig.xml'
-- **false**: replaces the parameters got from the corresponding msi parameters entry in ADT config file 'AppDeployToolkitConfig.xml'
+- **true** : appends the parameters to this one get from the corresponding parameters entry (if any) in ADT config file 'AppDeployToolkitConfig.xml'
```

Comment:
The whole sentance should be refactored.

E.g.
appends these paramterers to the defaults (if any) defined in the ADT config file 'AppDeployToolkitConfig.xml'